File: python/taller_than_meosseug.py
# 머쓱이보다 키 큰 사람

'''
Answer 1

def solution(array, height):
    answer = 0
    
    for num in array:
        if num > height:
            answer += 1

    return answer

'''


'''
Answer 2

def solution(array, height):
    return sum(1 for a in array if a > height)

'''

File: cpp/taller_than_meosseug.cpp
// 머쓴이보다 키 큰 사람

#include <string>
#include <vector>

using namespace std;

/* Answer 1
int solution(vector<int> array, int height) {
    int answer = 0;
    for(int i = 0; i < array.size(); i++)
    {
        if(array[i] > height)
        {
            answer++;
        }
    }   
    return answer;
} */

/* Answer 2
  
int solution(vector<int> array, int height) {
    int answer = 0;
    for(auto num : array)
    {
        if(num > height)
            answer++;
    }
    return answer;
} */
